docs/release-notes/8.14.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[release-notes-header-8.14.0]]
+== 8.14
+
+[discrete]
+[[release-notes-8.14.0]]
+=== 8.14.0
+
+[discrete]
+[[features-8.14.0]]
+==== New features
+* Introduces Attack discovery, a new feature that uses AI to identify potential attacks and help you quickly triage multiple alerts ({kibana-pull}181818[#181818]).
+* Creates the **Asset criticality** page within the **Manage** menu, which lets you bulk assign asset criticality levels to your assets ({kibana-pull}179891[#179891]).
+* Adds alert suppression for New Terms rules ({kibana-pull}178294[#178294]).
+* Adds alert suppression for EQL rules with non-sequence queries ({kibana-pull}176422[#176422]).
+* Allows you to edit value lists from the UI, anywhere you use them ({kibana-pull}179339[#179339]).
+* Adds a **Setup guide** markdown field to custom rules ({kibana-pull}178131[#178131]).
+
+[discrete]
+[[enhancements-8.14.0]]
+==== Enhancements
+* Removes the "Technical preview" tag for {esql}; it is now generally available ({kibana-pull}180838[#180838]).
+* Allows you to add calculated values to an {esql} rule's highlighted fields ({kibana-pull}177746[#177746]).
+* Connects {esql} functionality in {elastic-sec} to the `general:enableESQL` advanced setting ({kibana-pull}181616[#181616]).
```

Comment:
Can we bundle these together? E.g., `The following {esql} enhancements have been added in 8.14:` or something to that effect.

Reply:
Hmm, even if 181616 modifies existing ES|QL functionality in Security, I think it's significant enough to warrant being documented separately. The PR essentially allows users to turn ES|QL on/off throughout the entire Security app, wherever ES|QL is used.